src/components/Accordion/PAccordion.vue

```diff
@@ -0,0 +1,91 @@
+<template>
+  <div class="p-accordion">
+    <template v-for="section in sections" :key="section">
+      <div class="p-accordion__section">
+        <div class="p-accordion__header">
+          <slot :name="`${section}-heading`">
+            <slot name="heading" :section="section">
+              {{ section }}
+            </slot>
+          </slot>
+          <p-button
+            size="sm"
+            icon="ChevronDownIcon"
+            flat
+            rounded
+            class="p-accordion__button"
+            :class="getButtonClass(String(section))"
+            @click="toggleSection(String(section))"
+          />
+        </div>
+
+        <PAutoHeightTransition>
+          <template v-if="isOpen(String(section))">
+            <div class="p-accordion__content">
+              <!-- this is needed because PAutoHeightTransition does not animate an elements padding correctly -->
+              <div class="p-accordion__content-padding">
```

Comment:
Maybe a `wrapper` and `content` naming convention would make this more BEMy?

Reply:
Both follow BEM. That second div is only necessary because of the transition bug. So I think an overly specific class helps keep that div from being used for anything else so that it can hopefully be removed in the future.